lesson_2/Section_2.4.ipynb

```diff
@@ -0,0 +1,362 @@
+{
```

Comment:
<p>A lot of sentences are missing . at the end.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>